src/design/themes.stories.mdx

```diff
@@ -32,6 +33,30 @@ Optimized for page intros and headings.
   </Story>
 </Canvas>
 
+## Alternate
+
+An alternate, or secondary, theme that can be applied to sections.
```

Comment:
```suggestion
An alternate, or secondary, theme that can offset a section of content visually from its surroundings. This may be helpful for distinguishing rows of content from one another in a subtle way.
```